.github/workflows/opentelemetry.yml

```
@@ -0,0 +1,19 @@
+name: OpenTelemetry Export Trace
+
+on:
+  workflow_run:
+    workflows: [Test]
+    types: [completed]
+
+jobs:
+  otel-export-trace:
+    name: OpenTelemetry Export Trace
+    runs-on: ubuntu-latest
+    steps:
+      - name: Export Workflow Trace
+        uses: inception-health/otel-export-trace-action@latest
```

The existing workflow does not support to set the name of the service, so I'll change it to use the open PR https://github.com/inception-health/otel-export-trace-action/pull/13

```suggestion
        uses: v1v/otel-export-trace-action@v2
```